zenoh-flow/src/runtime/dataflow/instance/runners/connector.rs

```diff
@@ -122,72 +138,84 @@ impl Node for ZenohSender {
+                                )
+                            })?
+                        }
+                    };
 
-                // Getting the underlying slice in the shared memory
-                let slice = unsafe { buff.as_mut_slice() };
+                    // Getting the underlying slice in the shared memory
+                    let slice = unsafe { buff.as_mut_slice() };
 
-                // WARNING ACHTUNG ATTENTION
-                // This may fail as the message could be bigger than
-                // the shared memory buffer that was allocated.
-                match message.serialize_bincode_into(slice) {
-                    Ok(_) => {
-                        // If the serialization is success then we send the
-                        // shared memory buffer.
-                        self.z_session
-                            .put(self.key_expr.clone(), buff)
-                            .congestion_control(CongestionControl::Block)
-                            .res()
-                            .await
-                    }
-                    Err(e) => {
-                        // Otherwise we log a warn and we serialize on a normal
-                        // Vec<u8>
-                        let data = message.serialize_bincode()?;
-                        log::warn!(
-                            "[ZenohSender: {}] Unable to serialize into shared memory: {}, serialized size {}, shared memory size {}",
-                            self.id,
-                            e,
-                            data.len(),
-                            self.shm_element_size,
-                        );
+                    // WARNING ACHTUNG ATTENTION
+                    // This may fail as the message could be bigger than
+                    // the shared memory buffer that was allocated.
+                    match message.serialize_bincode_into(slice) {
+                        Ok(_) => {
+                            // If the serialization is success then we send the
+                            // shared memory buffer.
+                            self.z_session
+                                .put(self.key_expr.clone(), buff)
+                                .congestion_control(CongestionControl::Block)
+                                .res()
+                                .await
+                        }
+                        Err(e) => {
+                            // Otherwise we log a warn and we serialize on a normal
+                            // Vec<u8>
+                            let data = message.serialize_bincode()?;
+                            log::warn!(
+                                "[ZenohSender: {}] Unable to serialize into shared memory: {}, serialized size {}, shared memory size {}",
+                                self.id,
+                                e,
+                                data.len(),
+                                self.shm_element_size,
+                            );
```

Comment:
It seems that you have introduced tabs instead of spaces here for indentation

Reply:
Pff, stupid VS Code